plugins/connection/oc.py

```diff
@@ -85,7 +85,7 @@
         vars:
           - name: ansible_oc_context
         env:
-          - name: k8S_AUTH_CONTEXT
+          - name: K8S_AUTH_CONTEXT
```

Comment:
I'm wondering why this change is shown again.

Reply:
I based it off the old branch without pulling the latest upstream master... I thought git/github would be smart enough to resolve that (it would on merge, I think).  Now I'm paranoid, though... /me updates master and rebases.

Reply:
I was thinking that github would be smart enough too... That's why I was so confused when I saw the change again. But apparently it is not.